File: app/src/main/java/com/talktiva/pilot/model/Count.java
package com.talktiva.pilot.model;

import com.google.gson.annotations.SerializedName;

import java.io.Serializable;

public class Count implements Serializable {

    @SerializedName("eventCount")
    private Integer eventCount;

    public Integer getEventCount() {
        return eventCount;
    }
}

File: app/src/main/java/com/talktiva/pilot/model/Count.kt
package com.talktiva.pilot.model

import com.google.gson.annotations.SerializedName
import java.io.Serializable

class Count : Serializable {

    @SerializedName("eventCount")
    var eventCount: Int? = null

    @SerializedName("userCount")
    var userCount: Int? = null

    @SerializedName("emailVerified")
    var emailVerified: Boolean? = null

    @SerializedName("addressVerified")
    var addressVerified: Boolean? = null

    @SerializedName("addressProofUploaded")
    var addressProofUploaded: Boolean? = null
}
